utilities/src/main/java/io/syndesis/qe/resource/impl/Syndesis.java

```diff
@@ -66,7 +58,30 @@ public static void deploy() {
         TodoUtils.createDefaultRouteForTodo("todo2", "/");
     }
 
-    private static void createPullSecret() {
+    @Override
+    public void undeploy() {
+        // Intentionally left blank
```

Comment:
Ok, but maybe explain why? Also, write that in the log so we prevent hair pulling in the future :D

Reply:
I wanted to use current behavior - syndesis will stay deployed after the test everytime, but other stuff are always undeployed :grin: 

Doesn't make much sense, but that is how it is right now.. any idea? Maybe re-introduce `syndesis.config.namespace.cleanup.after`? :grin:

Reply:
Right, I meant explain in the comments for when someone reads this code in the future.